src/Akka.Logger.Serilog/SerilogLoggingAdapterExtensions.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using Akka.Actor;
+using Akka.Event;
+
+namespace Akka.Logger.Serilog
+{
+    public static class SerilogLoggingAdapterExtensions
+    {
+        /// <summary>
+        /// Create a logger that enriches log events with the specified property.
+        /// </summary>
+        /// <param name="adapter">ILoggingAdapter instance</param>
+        /// <param name="propertyName">The name of the property. Must be non-empty.</param>
+        /// <param name="value">The property value.</param>
+        /// <param name="destructureObjects">If true, the value will be serialized as a structured object if possible; if false, the object will be recorded as a scalar or simple array.</param>
+        public static ILoggingAdapter ForContext(this ILoggingAdapter adapter, string propertyName, object value, bool destructureObjects = false)
+        {
+            var customAdapter = adapter as SerilogLoggingAdapter;
+            return customAdapter == null ? adapter : customAdapter.SetContextProperty(propertyName, value, destructureObjects);
+        }
+
+        /// <summary>
+        /// Creates a new logging adapter using the specified context's event stream.
+        /// </summary>
+        /// <param name="context">The context used to configure the logging adapter.</param>
+        /// <returns>The newly created logging adapter.</returns>
+        public static ILoggingAdapter GetLogger<T>(this IActorContext context)
+            where T : class, ILoggingAdapter
+        {
+            var logSource = context.Self.ToString();
+            var logClass = context.Props.Type;
+
+            return Activator.CreateInstance(typeof(T), new BusLogging(context.System.EventStream, logSource, logClass, new SerilogLogMessageFormatter())) as T;
```

Comment:
Ofcourse. But API wise, you'd get a conflic with the normal context.GetLogger(). And then your still manually using the BusLogging class. Which (even though its public) is an internal implementation aspect of Akka.Net imo.

Thats whats my comment from 2 days ago is about. So its not something you can fix in this PR. Because it needs changes in the Core of Akka.Net.